bears/general/MementoBear.py

```diff
@@ -65,11 +79,16 @@ def run(self, filename, file, dependency_results=dict(),
         :param dependency_results: Results given by URLBear.
         :param follow_redirects:   Set to true to check all redirect urls.
         """
+        self.default_url_ignore = \
```

Comment:
As the user can not modify this value, it should be private, and not called "default".  So simply `_timegate_uris`.

And the bear documentation (docstring) should mention that timegate URIs are automatically not checked.

Reply:
updated